src/javascript/static/pages/contact.js

```diff
@@ -1,17 +1,44 @@
-const Dropdown = require('@binary-com/binary-style').selectDropdown;
+const moment           = require('moment');
+const Dropdown         = require('@binary-com/binary-style').selectDropdown;
+const setExternalTimer = require('../../app/base/clock').setExternalTimer;
 
 const Contact = (() => {
+    let $chat_button,
+        $chat_unavailable;
 
     const onLoad = () => {
+        $chat_button      = $('#chat_button');
+        $chat_unavailable = $('#live_chat_unavailable');
+        showHideLiveChat();
+        setExternalTimer(showHideLiveChat);
```

Comment:
Not sure if it's a good idea to call this every second.
How about a setTimeout with the remaining time till the next state?

Reply:
Done, tried to make it as clear as possible, let me know if it can be improved.